plugins/go-build/share/go-build/1.14.8

```diff
@@ -0,0 +1,11 @@
+install_darwin_64bit "Go Darwin 64bit 1.14.8" "https://golang.org/dl/go1.14.8.darwin-amd64.tar.gz#6fbfdca4e876477f1cb54df09b310e95f335e1d49556fe5c75bca02159b94c63"
```

Comment:
could be https://dl.google.com/go/go1.14.8.darwin-amd64.tar.gz#6fbfdca4e876477f1cb54df09b310e95f335e1d49556fe5c75bca02159b94c63 ?
ref: https://github.com/syndbg/goenv/issues/131